frame/multisig/src/lib.rs

```diff
@@ -73,6 +74,20 @@ pub use weights::WeightInfo;
 
 pub use pallet::*;
 
+/// The log target of this pallet.
+pub const LOG_TARGET: &'static str = "runtime::multisig";
+
+// syntactic sugar for logging.
+#[macro_export]
+macro_rules! log {
```

Comment:
Why is this in here if you only use it in the migration?

Reply:
Because I'd rather define logging macro for the whole package and have the opportunity to use it when/if needed. It's also consistent with other pallets which define logging.

Reply:
@ruseinov's point is fair to me, I had to do the same in the past.

@ggwpez what I challenge you about is a way so we won't need to define this log macro per-pallet? wdyt?